packages/contentful/migrations/0143-create-features-page-content-type.cjs

```diff
@@ -0,0 +1,140 @@
+module.exports = function(migration) {
+  const voteableFeature = migration
+    .createContentType('voteableFeature')
+    .name('Feature')
+    .description(
+      'A feature that users can vote on.'
+    )
+    .displayField('name');
+
+  voteableFeature
+    .createField('name')
+    .name('Title')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([{ size: { max: 100 },
+      message: 'Text must be max. 100 characters.' }])
+    .disabled(false)
+    .omitted(false);
+
+  voteableFeature
+    .createField('identifier')
+    .name('identifier')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        unique: true
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  voteableFeature
+    .createField('text')
+    .name('Text')
+    .type('Text')
+    .localized(true)
+    .required(false)
+    .validations([{ size: { max: 300 },
+      message: 'Text must be max. 300 characters.' }])
+    .disabled(false)
+    .omitted(false);
+
+  voteableFeature
+    .createField('image')
+    .name('Image')
+    .type('Link')
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        linkContentType: ['illustration']
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Entry');
+
+  voteableFeature.changeFieldControl('text', 'builtin', 'markdown', {});
+
+  const featureIdeasPage = migration
+    .createContentType('featureIdeasPage')
+    .name('Feature ideas page')
+    .description(
+      'The Feature ideas page.'
+    )
+    .displayField('name');
+
+  featureIdeasPage
+    .createField('name')
+    .name('Title')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([
+      {
+        unique: true
+      },
+      {
+        size: {
+          max: 100
+        },
+
+        message:
+          'This is an H1 field, it needs to be unique for SEO and not more than 100 characters.'
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  featureIdeasPage
+    .createField('description')
+    .name('Description')
+    .type('Symbol')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  featureIdeasPage.changeFieldControl('description', 'builtin', 'singleLine', {
+    helpText: 'For SEO, please make sure there is a short description.'
```

Comment:
This is not just for SEO though, it's also what's displayed at the top of the page.

The description from the design already doesn't fit into the limit. If we want to ensure there is a short(er) SEO description perhaps we should split out another field for the displayed description which could then also use the rich-text editor for more flexibility down the line.

Reply:
This description is meant to be only for SEO. The intro text on the design is indeed too long to be a description, and is also displayed below the 'content header' (below the share button), styled as a static page text section. So I modeled it as a rich text section. Under sections you can add a rich text section which should be used for the introduction text.

Reply:
We could split it into a separate field on the page level, instead of as a section.

Reply:
yes, let's do that. Otherwise editors could add multiple richtexts, with only one showing as the description.